packages/db-model/src/utils/pagination.ts

```diff
@@ -0,0 +1,17 @@
+/**
+ * Build the pages for PaginationResult
+ *
+ * @param {number} page
+ * @param {number} pageSize
+ * @param {number} count
+ * @returns {Object}
+ */
+export function buildPages(page: number, pageSize: number, count: number) {
```

Comment:
This will make the editor experience better. 
```ts
interface PaginationResult {
  first: number;
  prev: number;
  current: number;
  next: number;
  last: number;
}
```
```suggestion
 * @returns {PaginationResult}
 */
export function buildPages(page: number, pageSize: number, count: number): PaginationResult {
```